style_editor.html

```diff
@@ -5,6 +5,9 @@
   <meta charset="UTF-8">
   <!-- This should show up in search engine results. -->
   <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
+  <link rel="stylesheet" href="fontawesome-free-6.4.0-web/css/all.css" crossorigin="anonymous"/>
```

Comment:
I took this out because the style editor wouldn't work with a style URL with it.
Does it work now?

Reply:
Yes, tested just now and working. Not sure why a URL would matter.

Reply:
That's some voodoo I couldn't deconstruct. Best guess was it is for the history tab.
Commenting the whole magic css out, with the additions, I saw no difference on the rest of things.